### PR TITLE
Update updates.rst

### PR DESCRIPTION
**Pull request by cotosso**
updates page: update to match implementation

### CODE REVIEW

File: updates.rst [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
+Typically they are performed automatically, but at any time it is possible to check for new updates available by clicking on :guilabel:`Check for Fixes` button.
+These updates do not require a restart of NethSecurity, they are tied to a specific version and distributed using packages.
 
-The ``opkg update`` command updates the list of available packages, and the ``opkg upgrade`` command upgrades all of the installed packages to the latest versions.
+**2. System updates** (switching to a different version)
 
-**2. New features are released with new images.** To update the system to a new version with new features, download the new image file and run the following command: ::
+These updates involve the transition to a new version of the firmware that introduces new features, improvements and a greater hardware support.
 
-  sysupgrade -v nethsecurity-<version>-x86-64-generic-squashfs-combined.img.gz
```
Comment (filippocarletti) [inline updates.rst:19]: Are we sure we want to remove this information? It may be useful.

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: updates.rst (hunk 1 above)
Comment (cotosso) [reply updates.rst:19]: might it be better to put it in the dev manual instead of this manual?
https://dev.nethsecurity.org

Comment (filippocarletti) [reply updates.rst:19]: It's useful to users, not to developers.

Comment (cotosso) [reply updates.rst:19]: I have nothing against it

File: updates.rst [hunk 2]
```diff
@@ -4,20 +4,30 @@
...
+This type of update will reboot the device (which will therefore not be reachable for a few dozen seconds) and then completely rewrites the firmware, preserving all the configurations.
+However it is recommended to save a configuration backup before proceeding with the upgrade.
+
+If a new version is available, the user interface will display an information banner and a dedicated button :guilabel:`Update System` that will allow you to perform the update.
+
+Alternatively, it is always possible to manually upload a compatible image using the :guilabel:`Update with image file` button and proceed with the update.
+
+.. note:: You can also perform a ``System update`` from the command line.
+
+To do this, simply save download the new image file (it is recommended to save it in /tmp) and run the following command: ::
```
Comment (gsanchietti) [inline updates.rst:29]: The not block syntax was broken:

```suggestion
You can also perform a ``System update`` from the command line.
To do this, simply save download the new image file (it is recommended to save it in /tmp) and run the following command: ::
```

**Review by gsanchietti — APPROVED** [review]

File: updates.rst [hunk 3]
```diff
@@ -4,20 +4,29 @@
...
+NethSecurity allows two types of updates, both available from the ``Update`` section under the ``System`` menu.
 
-NethSecurity uses two different methods to update the system:
+**1. Bug & security fixes**
 
-**1. Security and minor bugfixes are distributed using packages.** To update the system, run the following commands: ::
+These updates are intended for minor updates and bugfixes.
 
-    opkg update && opkg upgrade
+Typically they are performed automatically, but at any time it is possible to check for new updates available by clicking on :guilabel:`Check for Fixes` button.
```
Comment (gsanchietti) [inline updates.rst:13]:
```suggestion
Typically they could be performed automatically, but at any time it is possible to check for new updates available by clicking on :guilabel:`Check for Fixes` button.
```

**Pull request merged**